aptos-move/framework/aptos-stdlib/sources/cryptography/bls12381.move

```diff
@@ -19,9 +21,17 @@ module aptos_std::bls12381 {
     /// One of the given inputs has the wrong size.s
     const EWRONG_SIZE: u64 = 2;
 
+    /// The number of signers does not match the number of messages to be signed.
+    const ESIGNER_COUNT_NOT_MATCH_MESSAGE_COUNT: u64 = 3;
```

Comment:
Maybe rename to slightly shorter `E_NUM_SIGNERS_MUST_EQ_NUM_MESSAGES`? 

Otherwise, you'd have to rename `ESIGNER_COUNT_NOT_MATCH_MESSAGE_COUNT` to add a `**DOES**_NOT`